docs-ui/components/panels.stories.js

```diff
@@ -90,6 +120,42 @@ export const _PanelTable = withInfo({
       <div>Panel Item</div>
       <div>Panel Item</div>
     </PanelTable>
+
+    <BulkController
```

Comment:
Can you make this into its own story? Otherwise it is hard to find

Reply:
I had it like that initially but got feedback from the design team that it should live next to other tables. Would it make sense to have own bulkController story that we could see in the sidebar, but also call that component at the bottom of all PanelTables? (see https://github.com/getsentry/sentry/pull/20501/commits/84aa84fd2d2a8cf0f659c465fed7ad1a623ded2d)

Reply:
Yeah I think that's good.

Reply:
Main problem was being able to use the search to filter for it